validator/client/validator.go

```diff
@@ -661,6 +662,7 @@ func (v *validator) logDuties(slot types.Slot, duties []*ethpb.DutiesResponse_Du
 			} else {
 				proposerKeys[proposerIndex] = validatorKey
 			}
+			ValidatorNextProposalSlotGaugeVec.WithLabelValues(validatorNotTruncatedKey).Set(float64(proposerSlot))
```

Comment:
This needs to be wrapped with if v.emitAccountMetrics {}
```suggestion
if v.emitAccountMetrics {			 
  ValidatorNextProposalSlotGaugeVec.WithLabelValues(validatorNotTruncatedKey).Set(float64(proposerSlot))
}
```